apps/api/v2/src/modules/oauth/user/oauth-user.e2e-spec.ts

```diff
@@ -0,0 +1,180 @@
+import { bootstrap } from "@/app";
```

Comment:
should we add .controller in this file name ?

Reply:
oauth-user.controller.e2e-spec.ts

Reply:
I like the idea of this convention! Done in [rename test file](https://github.com/calcom/cal.com/pull/12853/commits/cd318586bbe87d8a074e6724f64dac906e9822f1)